Classes_Homework5/__main__.py

```diff
@@ -0,0 +1,24 @@
+from exec_utils.configloader import Config
+from modules.input import Selection
+from modules.run import Execute
+
+cnf = Config()
+s = Selection()
+e = Execute()
+
+msg1, msg2 = cnf.get_values("INPUTS", "proceed_msg"), cnf.get_values("MESSAGES", "goodbye")
+
+
+def main(msg1, msg2):
+    """Run end-to-end flow (ask for input, write to file),
+    ask if user wants to proceed - if yes, run again, if no - print goodbye message.
+    Both messages are configurable, see in configs.ini"""
+    decision = "yes"
+    while decision == "yes":
```

Comment:
This "while" loop looks good. However, a little bit more Pythonic way for such task:
```
while True:
    <your_code>
    break;
```